packages/types/tsconfig.json

```diff
@@ -2,5 +2,6 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "build"
-  }
+  },
+  "exclude": ["build"]
```

Comment:
Do we also need node_modules here?

Reply:
There is no node_modules in types package

Reply:
I think I'd still add this a) for consistency, but also b) because if some dependencies are added node_modules might appear and break builds